composer.json

```diff
@@ -138,8 +141,11 @@
             "Tempest\\Mapper\\Tests\\": "src/Tempest/Mapper/tests",
             "Tempest\\Reflection\\Tests\\": "src/Tempest/Reflection/tests",
             "Tempest\\Router\\Tests\\": "src/Tempest/Router/tests",
-            "Tempest\\Support\\Tests\\": "src/Tempest/Support/tests",
+            "Tempest\\Support\\Tests\\": [
+                "src/Tempest/Support/tests"
+            ],
```

Comment:
I'm guessing this was `composer merge`. Do we know why it did this?

Reply:
It is indeed `composer merge`. I does that because of the square brackets, which are not required, not sure why they're here